sdcm/report_templates/results_gemini.html

```diff
@@ -43,14 +22,79 @@ <h3>
         </ul>
     </div>
     {% endif %}
-{% endblock %}
+    <h3>Nemesis Details</h3>
+    <h4>Running nemesis: {{ nemesis_name }}</h4>
```

Comment:
this is part of the `results_longevity` template. Why did you copy-paste it?

Reply:
I think we can fully use the longevity template. Is there any reason to see this table in the email body and not in the attached HTML?

Reply:
I use it to have the same view for email body and for attached html for gemini with nemesis. Email body will have short variant, and attached html will have full report
yes i want to reached this but we need also to have the gemini stats.